tests/foreman/cli/test_host.py

```diff
@@ -2012,10 +2012,15 @@ def test_negative_without_attach(
     assert "This system has no repositories available through subscriptions." in repo_list.stdout
 
 
+@pytest.mark.parametrize('sca_enabled', [True, False], ids=['with_sca', 'without_sca'])
 @pytest.mark.cli_host_subscription
 @pytest.mark.tier3
 def test_negative_without_attach_with_lce(
-    target_sat, host_subscription_client, function_org, function_lce
+    target_sat,
+    host_subscription_client,
+    function_org,
+    function_lce,
+    sca_enabled,
 ):
     """Attempt to enable a repository of a subscription that was not
```

Comment:
I think we should update the description wrt to SCA switch and expected results. Also good to have test name modified for sca switching!

Reply:
Right. I'll update `:expectedresults:` to reflect what we expect to happen with{,out} SCA.

I wouldn't change the test name though in order to keep history. I think it's descriptive enough as it is.